modules/bene_media_file/config/rewrite/core.entity_form_display.media.file.default.yml

```diff
@@ -1,43 +1,50 @@
+config_rewrite: replace
 langcode: en
 status: true
 dependencies:
   config:
-    - field.field.media.image.image
-    - field.field.media.image.media_tags
-    - image.style.thumbnail
-    - media_entity.bundle.image
+    - field.field.media.file.field_media_file
+    - field.field.media.file.media_tags
+    - media.type.file
   module:
-    - image
-id: media.image.default
+    - file
+id: media.file.default
 targetEntityType: media
-bundle: image
+bundle: file
 mode: default
 content:
-  image:
-    type: image_image
+  field_media_file:
     weight: 0
     settings:
       progress_indicator: throbber
-      preview_image_style: thumbnail
     third_party_settings: {  }
+    type: file_generic
     region: content
   media_tags:
-    weight: 4
+    type: entity_reference_autocomplete
```

Comment:
I like `entity_reference_autocomplete_tags` better than autocomplete.

![screen shot 2018-02-26 at 9 59 35 am](https://user-images.githubusercontent.com/999525/36686601-c5d5a8d6-1adb-11e8-97d4-ece52ce29038.png)

Reply:
Agreed. Whoever gets to it first should change this.